packages/@okta/vuepress-site/docs/release-notes/2022/index.md

```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
 
 ## July
 
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bug fixed in 2022.07.1](#bug-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set `issuerMode` to `DYNAMIC`, Okta uses the domain from the `authorizeURI` as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```

Comment:
```suggestion
You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set  the `issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the `authorizeURI` property as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Where is `authorizeURI` property coming from? (which object?) I don't see this property in the IdP API. Perhaps we need to add references to this property?
Also, is PR 3469 going out with this release?

Reply:
Updated your other suggestions locally.